common/app/common/configuration.scala

```diff
@@ -308,7 +308,6 @@ class GuardianConfiguration(val application: String, val webappConfDirectory: St
     lazy val masterclassesToken = configuration.getStringProperty("masterclasses.token")
     lazy val liveEventsToken = configuration.getStringProperty("live-events.token")
     lazy val liveEventsImagesUrl = "https://membership.theguardian.com/events.json"
-    lazy val jobsUrlTemplate = configuration.getStringProperty("jobs.api.url.template")
     lazy val jobsStaticUrl= configuration.getStringProperty("jobs.api.url")
```

Comment:
A small point but could this just be `jobsUrl`?
The fact that it's static doesn't seem to make any difference to how it's used.